spec.html

```diff
@@ -25959,6 +25971,22 @@ <h1>Properties of AggregateError Instances</h1>
         <p>AggregateError instances are ordinary objects that inherit properties from their AggregateError prototype object and have an [[ErrorData]] internal slot whose value is *undefined*. The only specified use of [[ErrorData]] is by `Object.prototype.toString` (<emu-xref href="#sec-object.prototype.tostring"></emu-xref>) to identify Error, AggregateError, or _NativeError_ instances.</p>
       </emu-clause>
     </emu-clause>
+
+    <emu-clause id="sec-errorobjects-abstracts">
+      <h1>Abstract Operations for Error Objects</h1>
+
+      <emu-clause id="sec-errorobjects-install-error-cause">
+        <h1>InstallErrorCause ( _O_, _options_ )</h1>
+        <p>The abstract operation InstallErrorCause takes arguments _O_ and _options_. It is used to create `"cause"` property on _O_ when the `"cause"` property is present on _options_. It performs the following steps when called:</p>
+        <emu-alg>
+          1. Assert: Type(_O_) is Object.
+          1. If Type(_options_) is Object and ? HasProperty(_options_, `"cause"`) is *true*, then
```

Comment:
```suggestion
          1. If Type(_options_) is Object and ? HasProperty(_options_, *"cause"*) is *true*, then
```
and likewise for the other 2 occurrences.